Sources/NIO/Selector.swift

```diff
@@ -292,8 +324,8 @@ final class Selector<R: Registration> {
     ///
     /// - parameters:
     ///     - selectable: The `Selectable` to re-register.
-    ///     - interested: The `IOEvent`s in which we are interested and want to be notified about.
-    func reregister<S: Selectable>(selectable: S, interested: IOEvent) throws {
+    ///     - interested: The `SelectorEventSet` in which we are interested and want to be notified about.
+    func reregister<S: Selectable>(selectable: S, interested: SelectorEventSet) throws {
```

Comment:
This should presumably also assert for `.reset`?

Reply:
done